data_warehouse/tests/tests_services.py

```diff
@@ -33,5 +33,47 @@ def test_dispatch_error(self):
     def test_dispatch(self):
         '''Should get a aggregated data'''
         self.method_name = 'staff_statistics'
+        self.context = {'request': self.request}
+        self.context['group_by'] = '0'
         AggregateDataService.dispatch(
             self.method_name, self.context)
+
+    def test_staff_statistics(self):
+        '''Should get a staff_statistics data'''
+        self.context = {'request': self.request}
+        with self.assertRaisesMessage(BadRequest, '错误的参数'):
+            AggregateDataService.staff_statistics(self.context)
+        self.context['group_by'] = '100'
+        with self.assertRaisesMessage(BadRequest, '错误的参数'):
+            AggregateDataService.staff_statistics(self.context)
+        self.context['group_by'] = '0'
+        self.context['region'] = '0'
+        AggregateDataService.staff_statistics(self.context)
+        self.context['group_by'] = '1'
+        AggregateDataService.staff_statistics(self.context)
+        self.context['group_by'] = '2'
+        AggregateDataService.staff_statistics(self.context)
+        self.context['group_by'] = '3'
+        AggregateDataService.staff_statistics(self.context)
```

Comment:
You are not testing anything here, invoking doesn't mean testing, check your output

Reply:
Ditto to all occurrences.